source/wp-content/themes/wporg-news-2021/sass/components/local-header/_breadcrumbs.scss

```diff
@@ -0,0 +1,33 @@
+body .wp-site-blocks .local-header {
+    // Breadcrumbs show the lineage of pages based on the current
+    // page. It contains a link for each parent.
+    .local-header__breadcrumb {
```

Comment:
Can you simplify these selectors (in each file), or do they need to be this nested for specificity? `.local-header__breadcrumb` by itself seems like it would target the correct thing.

Reply:
Ah yeah, good spot. Looks like they can all be simplified. I've removed `body .wp-site-blocks .local-header` from each, apart from `local-header.scss` which I've changed to `.local-header`.

Reply:
Ah wait, I've just spotted an issue on the archive page with some conflicting margins in the breadcrumbs. Fixing..

Reply:
Looks like it was just the breadcrumbs that needed more specificity, so I've added `.local-header` there as well.